x-pack/plugins/observability_solution/slo/public/pages/slos/components/card_view/slo_card_item.tsx

```diff
@@ -43,27 +40,6 @@ export interface Props {
   refetchRules: () => void;
 }
 
-export const useSloCardColor = (status?: SLOWithSummaryResponse['summary']['status']) => {
-  const colors = {
-    DEGRADING: useEuiBackgroundColor('warning'),
-    VIOLATED: useEuiBackgroundColor('danger'),
-    HEALTHY: useEuiBackgroundColor('success'),
-    NO_DATA: useEuiBackgroundColor('subdued'),
-  };
-
-  return { cardColor: colors[status ?? 'NO_DATA'], colors };
-};
```

Comment:
Moved to x-pack/plugins/observability_solution/slo/public/pages/slos/components/card_view/hooks/use_slo_card_colors.ts and use correct colors.background tokens